### PR TITLE
Tab toggle component for the My Posts page

### PR DESCRIPTION
**Pull request by nlitwin**
## Description

**Issue:** #103 

This is the 1st of 2 PRs which fleshes out the `My Posts` page. This first PR includes just the tab toggle and dummy content for the posts. The next PR will make the appropriate queries to the backend and display posts with `<PostCard />`

## Subtasks

- [x] I have added this PR to the Journaly Kanban project ✅
- [x] Create `TabToggle` for the `my-posts` page
- [x] Add dummy content for `My Posts` and `Drafts`

## Screenshots

tab-toggle-my-posts

### CODE REVIEW

File: frontend/pages/dashboard/my-posts.tsx [hunk 1]
```diff
@@ -1,16 +1,51 @@
...
+import React, { useState } from 'react'
 import { NextPage } from 'next'
-
 import { withApollo } from '../../lib/apollo'
 import DashboardLayout from '../../components/Layouts/DashboardLayout'
+import TabToggle from '../../elements/TabToggle'
 
 const MyPostsPage: NextPage = () => {
+  const tabs = [
+    { key: 'my-posts', text: 'My Posts' },
```
Comment (robin-macpherson) [inline frontend/pages/dashboard/my-posts.tsx:9]: Can we change this to "Published"?

File: frontend/pages/dashboard/my-posts.tsx [hunk 2]
```diff
@@ -1,16 +1,51 @@
...
+    setActiveKey(key)
+  }
+
   return (
     <DashboardLayout>
-      <h1>My Posts</h1>
+      <TabToggle activeKey={activeKey} tabs={tabs} onToggle={handleToggle} />
+
+      <div className="posts-wrapper">
+        {activeKey === 'published' && <div>My Posts</div>}
```
Comment (robin-macpherson) [inline frontend/pages/dashboard/my-posts.tsx:24]: Does the text in the `<div>` need to be updated to "Published" as well? Is that what will show as the title?

Comment (robin-macpherson) [reply frontend/pages/dashboard/my-posts.tsx:24]: If it's the title I wonder if we could make it "Published Posts" and "Drafts

Comment (nlitwin) [reply frontend/pages/dashboard/my-posts.tsx:24]: These are just temporary divs that will be deleted and replaced with new components, probably something like `MyPosts.tsx` and `Drafts.tsx`. In the follow-up PR which adds these, I can add a heading/title above the posts 👍  `"My Published Posts"` and `"My Drafts"` sound good to me!

Comment (robin-macpherson) [reply frontend/pages/dashboard/my-posts.tsx:24]: Ah ok cool cool sounds good!

**Pull request merged**